TTS/bin/compute_embeddings.py

```diff
@@ -100,11 +72,20 @@
 
 if speaker_mapping:
     # save speaker_mapping if target dataset is defined
-    if ".json" not in args.output_path:
+    if '.json' not in args.output_path and '.npy' not in args.output_path:
```

Comment:
Do `.json` and `.npy` files have the same content?

Reply:
Yes, but the numpy takes up much less space. When working with large files this may be a better option than .json.

Reply:
Yeah makes sense.  But then you need to update the speaker manager to handle npy too to keep the coder running after your PR

Reply:
I removed the .npy save at this point, as in most cases space is not an issue.